.github/workflows/kind-e2e.yaml

```diff
@@ -35,18 +39,46 @@ jobs:
       run: |
         make toolchain
         make install-kwok
-        KWOK_REPO=kind.local KIND_CLUSTER_NAME=chart-testing make apply-with-kind
+        export KWOK_REPO=kind.local
+        export KIND_CLUSTER_NAME=chart-testing
+        make apply-with-kind
     - name: ping cluster
       shell: bash
       run: | 
+        sleep 15
         kubectl get pods -n kube-system | grep karpenter 
         kubectl get nodepools
         kubectl get pods -A
         kubectl describe nodes
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      with:
+        repository: nathangeology/karpenter_evaluate
+        path: ./karpenter_eval/ # Installs to a folder in the Karpenter repo for the test
+        ref: "3f4cebb703bd136f5034c8b5bf3e6c32e97d1ae6"
+        fetch-depth: 0
+    - name: install KPI report dependencies
+      shell: bash
+      run: |
+        pip install awswrangler
```

Comment:
Can we remove this in favor of a non-aws tool?

Reply:
Ok, I got this worked out and it seems to work ok.